Creating OCP performance guide (#564)
RedHatInsights/red-hat-ansible-automation-platform-documentation: downstream/assemblies/platform/assembly-specify-dedicated-nodes.adoc

ifdef::context[:parent-context: {context}]

:context: performance-considerations

[id="assembly-specify-dedicted-nodes"]
= Specifying dedicated nodes

A kubernetes cluster runs on top of multiple Virtual Machines or nodes (generally anywhere between 2 and 20 nodes). 
Pods can be scheduled on any of these nodes. 
When you create or schedule a new pod, use the `topology_spread_constraints` setting to configure how new pods are distributed across the underlying nodes when scheduled or created. 

> Typically your pods should not be scheduled on a single node, because if that node fails, so does the services those pods are providing.

Reviewer: ```suggestion
Do not schedule your pods on a single node, because if that node fails, the services that those pods provide fails.
```

> The control plane nodes be run on different nodes to the automation job pods. 

Reviewer: ```suggestion
Schedule the control plane nodes to run on different nodes to the automation job pods. 
```
